.github/workflows/lint.yml

```diff
@@ -3,12 +3,14 @@ name: Lint
 on:
   pull_request:
 
+env:
+  AMENT_CPPCHECK_ALLOW_SLOW_VERSIONS: 1
```

Comment:
just wondering: why is this needed?

Reply:
This is a workaround for the error below. https://github.com/ros-tooling/topic_tools/actions/runs/3209171467/jobs/5245668834
```
cppcheck 2.7 has known performance issues and therefore will not be used, set the AMENT_CPPCHECK_ALLOW_SLOW_VERSIONS environment variable to override this.
```

Reply:
oh okay, thanks for the explanation!